packages/react/src/Grid/gridCellClasses.test.ts

```diff
@@ -0,0 +1,65 @@
+const { gridCellClasses } = require('./gridCellClasses')
```

Comment:
I was ready to move them over, then noticed that I’m not entirely convinced. Functions are very suitable to be unit tested, maybe even more so than components. And if we’re going to rewrite these tests we’re going to test the exact same outputs, only with slightly more complicated inputs and component mounting.

Would you still prefer the tests under the umbrella of the component instead of with the function?

I did notice the tests were entirely missing for `PageGrid` and `GridCell`, so I did add the usual suspects.

Reply:
We discussed to keep tests in the component for functions that are only used by that component.